packages/tinybased/src/lib/search/generateSearched.tsx

```diff
@@ -0,0 +1,116 @@
+/* eslint-disable @typescript-eslint/ban-types */
+import { SearchParams, SearchResult } from '@lyrasearch/lyra';
+
+import { useCallback, useEffect, useState } from 'react';
+import { useValue } from 'tinybase/cjs/ui-react';
+import { ObjectToCellStringType, TableBuilder } from '../TableBuilder';
+import { TinyBased } from '../tinybased';
+import { OnlyStringKeys, TinyBaseSchema } from '../types';
+import { Searched } from './searched';
+
+const SEARCH_LAST_UPDATED_AT = (table: string | symbol | number) =>
+  `${String(table)}-last-updated-at`;
```

Comment:
Let's make this name a little bit more unique since we're just adding content into a user supplied TinyBase. There's a small chance this could conflict with stuff they're already storing.

Maybe `__internal__libname__${table}-last-updated-at` or something that will definitely not overlap